tests/apisDB/setup_environment.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug  5 17:09:13 2021
+
+@author: pkh35
+"""
+
+import os
+import yaml
+from sqlalchemy import create_engine
+import logging
+
+log = logging.getLogger(__name__)
```

Comment:
good use of a logging library, these are stronger than print statements, I will switch to these soon